transform/models/marts/geo_reference/_geo_reference__models.yml

```diff
@@ -0,0 +1,101 @@
+version: 2
+
+sources:
+  - name: building_footprints
+    database: raw_dev
+    schema: building_footprints
+    tables:
+      - name: california_building_footprints
+  - name: tiger_2022
+    database: raw_dev
+    schema: tiger_2022
+    tables:
+      - name: blocks
+      - name: places
+
+models:
+  - name: geo_reference__blocks_joined
+    description: |
+      This data table is a join of the TIGER shapefile, Blocks,
+      with the Microsoft Building Footprints data for the state of CA.
+    columns:
+      - name: release
+        description: The version of the data
+      - name: capture_dates_range
+        description: Each building footprint has a capture date tag from 2019-2020
+      - name: county_fips
+        description: 2020 Census county FIPS code
+      - name: tract
+        description: 2020 Census tract code
+      - name: block
+        description: 2020 Census tabulation block number
+      - name: geoid
+        description: >
+          Census block identifier; a concatenation of 2020 Census state FIPS code, 2020
+          Census county FIPS code, 2020 Census tract code, and 2020 Census block number
+      - name: name
+        description: >
+          2020 Census tabulation block name; a concatenation of 'Block'
+          and the tabulation block number
+      - name: maf_tiger_feature_class_code
+        description: MAF/TIGER Feature Class Code
+      - name: maf_tiger_feature_class
+        description: >
+          The lowest-order census defined statistical area. It is an area,
+          such as a city block, bounded primarily by physical features but
+          sometimes by invisible city or property boundaries. A tabulation
+          block boundary does not cross the boundary of any other geographic
+          area for which the Census Bureau tabulates data. The subtypes of
+          this feature are Count Question Resolution (CQR), current, and
+          tabulation census.
+      - name: urban_rural_area
+        description: >
+          Reserved for 2020 Census urban/rural indicator
+          (2020 Urban Areas are not yet defined)
+      - name: housing_unit_count
+        description: 2020 Census housing unit count
+      - name: pop_count
+        description: 2020 Census population count
```

Comment:
I don't think we should include these, per our earlier conversation (perhaps also the feature classes).

Update: looks like you've removed some of them from the SQL, but not the documentation

Reply:
when you say perhaps also the feature classes are you referring to `class_fips_code` and `class_fips`?

Reply:
I meant `maf_tiger_feature_class` and `maf_tiger_feature_class_code`. But I'm a bit agnostic there.